<?xml version="1.0"?>
<launch>
  <arg name="config" default="$(find ca_driver)/config/default.yaml" />
  <arg name="desc" default="true" />

  <node name="ca_driver" pkg="ca_driver" type="ca_driver" output="screen">
    <rosparam command="load" file="$(arg config)" />
    <param name="robot_model" value="ROOMBA_400" />
  </node>

  <!-- Robot description -->
  <include if="$(arg desc)" file="$(find ca_description)/launch/roomba_400.launch" />
</launch>
